dask/bag/text.py

```diff
@@ -57,10 +57,12 @@ def read_text(path, blocksize=None, compression='infer',
     --------
     from_sequence: Build bag from Python sequence
     """
+    storage_options = storage_options or {}
     if isinstance(path, (tuple, list, set)):
         blocks = sum([read_text(fn, blocksize=blocksize,
                       compression=compression, encoding=encoding, errors=errors,
-                      linedelimiter=linedelimiter, collection=False, **kwargs)
+                      linedelimiter=linedelimiter, collection=False,
+                      **storage_options)
```

Comment:
@mrocklin Hmm, it does not "negatively affect", but rather just does not follow the function definition, so it just breaks things. Here is just a minimized example of what is happening now:

```python
def read_text(path, ..., storage_options=None):
    blocks = [read_text(fn, **storage_options) ...]
```

while it should be

```python
def read_text(path, ..., storage_options=None):
    blocks = [read_text(fn, ..., storage_options=storage_options) ...]
```